test/parser.test.js

```diff
@@ -165,14 +158,13 @@ QUnit.module('m3u8s', function(hooks) {
       '#EXTINF:5,',
       '#COMMENT',
       'ex1.ts',
-      '#EXT-X-CUE-IN',
+      '#EXT-X-CUE-IN:',
       '#EXTINF:5,',
       'ex2.ts',
       '#EXT-X-CUE-IN:15',
       '#EXT-UKNOWN-TAG',
       '#EXTINF:5,',
       'ex3.ts',
-      '#EXT-X-CUE-IN=abc',
```

Comment:
Should this be kept as `#EXT-X-CUE-IN:abc`? Then we can keep the test mostly the same.

Reply:
That's true, but to my mind the matching assertion would be almost identical to the one on line 185. As the intent of the removed assertion was to to explicitly call out that a `:` was not required I think it's clearer to remove it entirely.

Reply:
👍🏻